src/System.Private.Reflection.Metadata.Ecma335/src/Resources/Strings.resx

```diff
@@ -0,0 +1,516 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The immutable array code some resource string usage. I can't find a way to use multiple resx files.